lib/fortnox/api/types/model.rb

```diff
@@ -18,7 +18,7 @@ def initialize(input_attributes)
         private
 
         def missing_keys(attributes)
-          non_nil_attributes = attributes.reject { |_, value| value.nil? }
+          non_nil_attributes = attributes.compact
```

Comment:
I'd be for folding this into the next line now, since it is such a simple to read line `attribute_keys = attributes.compact.keys`